packages/network/src/logic/tracker/InstructionCounter.ts

```diff
@@ -3,6 +3,8 @@ import { Status } from '../../identifiers'
 import { NodeId } from '../node/Node'
 
 export const COUNTER_UNSUBSCRIBE = -1
+// Used by the tracker to signal to nodes that they are alone in the topology
+export const COUNTER_LONE_NODE = -2
```

Comment:
Is there a reason we signal this via counter and not just an empty neighbor list?